test/fake_modules/coffee_script/index.coffee

```diff
@@ -0,0 +1,3 @@
+require 'foo'
+require('bar')
+require('baz');
```

Comment:
Is CoffeeScript compatible with ES6 `import` declaration? Could you please add a test case for it?